sdk/storage/test/test_base.cpp

```diff
@@ -132,10 +132,27 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
-  std::vector<uint8_t> ReadBodyStream(Azure::Core::Http::BodyStream* stream)
+  std::vector<uint8_t> ReadBodyStream(std::unique_ptr<Azure::Core::Http::BodyStream>& stream)
   {
-    std::vector<uint8_t> body(stream->Length(), '\x00');
-    stream->Read(&body[0], body.size());
+    auto s = stream->Length();
+    std::vector<uint8_t> body((size_t)s, '\x00');
+    // Read 15k at at time. warning C6262:  exceeds /analyze:stacksize '16384'.
+    {
+      constexpr uint64_t fixedSize = 1024 * 15;
+      uint8_t tempBuffer[fixedSize];
```

Comment:
Both Azure Core SDK and Azure Storage SDK are supposed to be portable. So we cannot assume that on every platforms with all kinds of hardware configuration, the stack size is at least 16KB.